### PR TITLE
Fix Loopring contract information

### PR DESCRIPTION
**Pull request by lucadonnoh**

### CODE REVIEW

File: packages/config/src/layer2s/loopring.ts [hunk 1]
```diff
@@ -265,21 +265,21 @@ export const loopring: Layer2 = {
...
+        'Contract used by the Prover to submit exchange blocks with zkSNARK proofs that are later processed and verified by the BlockVerifier contract. Its owner (ProxyOwner) can use this contract to give or revoke permissions to submit blocks and to open block submission to everyone.',
       ),
       discovery.getContractDetails(
         'DefaultDepositContract',
         'ERC 20 token basic deposit contract. Handles user deposits and withdrawals.',
       ),
       discovery.getContractDetails(
         'LoopringV3',
-        'Contract managing LRC staking for exchanges (One Loopring contract can manage many exchanges).',
+        'Contract managing LRC staking for exchanges (one Loopring contract can manage many exchanges). Its owner (ProxyOwner) can use this contract to set the forced withdrawal fee and change the Verifier address.',
```
Comment (michalsidzej) [inline packages/config/src/layer2s/loopring.ts:276]: I think this belongs to Permissioned Addresses section, WYT?

Comment (lucadonnoh) [reply packages/config/src/layer2s/loopring.ts]: done

File: packages/config/src/layer2s/loopring.ts [hunk 2]
```diff
@@ -256,6 +256,12 @@ export const loopring: Layer2 = {
       description:
         'The rollup owner can submit blocks, set rollup parameters and shutdown the exchange.',
     },
+    {
+      name: 'LoopringV3',
+      accounts: [discovery.getPermissionedAccount('ExchangeV3', 'loopringV3')],
+      description:
+        'Contract managing LRC staking for exchanges (one Loopring contract can manage many exchanges). Its owner (ProxyOwner) can use this contract to set the forced withdrawal fee and change the Verifier address.',
```
Comment (michalsidzej) [inline packages/config/src/layer2s/loopring.ts:263]: Sorry, I think my comment wasn't clear. I meant the second sentence `Its owner (ProxyOwner) can use this contract to set the forced withdrawal fee and change the Verifier address.` should be a part of ProxyOwner description in the permissions section. LoopringV3 should stay in contracts section IMO

**Review by michalsidzej — CHANGES_REQUESTED** [review]
Please resolve the above comment

File: packages/config/src/layer2s/loopring.ts (hunk 2 above)
Comment (lucadonnoh) [reply packages/config/src/layer2s/loopring.ts:263]: ohh ok will do

**Review by michalsidzej — APPROVED** [review]

**Pull request merged**